### PR TITLE
OSDOCS-1990: Expanded information about machine pools and autoscaling.

### PR DESCRIPTION
**Pull request by jonquilwilliams**
JIRA: https://issues.redhat.com/browse/OSDOCS-1990

Previews:

https://deploy-preview-32710--osdocs.netlify.app/openshift-rosa/latest/nodes/nodes/nodes-machinepools-autoscaling-about.html

https://deploy-preview-32710--osdocs.netlify.app/openshift-rosa/latest/rosa_cli/rosa-manage-objects-cli.html#rosa-create-machinepool_rosa-managing-objects-cli

https://deploy-preview-32710--osdocs.netlify.app/openshift-rosa/latest/rosa_cli/rosa-manage-objects-cli.html#rosa-edit-machinepool_rosa-managing-objects-cli

Peer reviewer: Please merge to branch/dedicated-4 - No CP needed.

### CODE REVIEW

File: modules/rosa-edit-objects.adoc [hunk 1]
```diff
@@ -160,14 +160,20 @@ $ rosa edit machinepool --cluster=<cluster_name> | <cluster_id> <machinepool_ID>
...
 |Specifies the maximum number of compute nodes when enabling autoscaling.
 
 |--min-replicas
 |Specifies the minimum number of compute nodes when enabling autoscaling.
 
 |--replicas
-|The number (integer) of machines for this machine pool.
+|Required when autoscaling is not configured. The number (integer) of machines for this machine pool.
+
+|--taints
```
Comment (xueli181114) [inline modules/rosa-edit-objects.adoc:175]: The taints and label edit will only work for the new created nodes of the machine pool. Should we clarify it? For example, there is a machinepool with 2 nodes, and then user edit the taints, the new taints won't affect the existing nodes. If user scale up the nodes number to 3. Then the new created node will have the edited taints.

Comment (jonquilwilliams) [reply modules/rosa-edit-objects.adoc:175]: Added the clarification to taints and labels. Thanks.

**Comment by 0kashi** [comment]
@jonquilwilliams - Reviewed and added comments about the first link to this doc: https://docs.google.com/document/d/1mE9qjd-cquYWYn0X1JO2HjxCeAGabRBE2qVu5VW1tmc/edit?usp=sharing

**Review by mikemckiernan — APPROVED** [review]
lgtm

**Pull request merged**